docs/Changelog.md

```diff
@@ -990,6 +990,43 @@ This version of the operator has been available since version 1 of the default O
 <dd>Constrain input and output types to float tensors.</dd>
 </dl>
 
+### <a name="DynamicSlice-1"></a>**DynamicSlice-1**</a>
```

Comment:
When we decide to remove it from experimental, it should replace slice, right?

Reply:
Yes, I think that makes sense, unless people want the specialized version with the slices as attributes for optimization